### ISSUE TITLE
Forcing tight newline-spacing in TOC

### ISSUE THREAD

**Issue by Gewerd-Strauss** [Aidurber/obsidian-plugin-dynamic-toc#37]
Hello,

I have a small question, as the code you use in your reame 

![image](https://user-images.githubusercontent.com/80548781/150311622-149e4186-17ab-4680-b12d-4a527d43cc55.png)

doesn't seem to work on my end. Fresh install, settings are the following:

![image](https://user-images.githubusercontent.com/80548781/150312019-55c8477c-7e2a-4631-9ff2-8620966e606b.png)

This is the result on my end:

![image](https://user-images.githubusercontent.com/80548781/150311748-b1663310-0c54-4cb6-86fe-639ac5be3bfa.png)

I'd like to condense it - because otherwhise this can easily become very large on certain notes. I don't need the newlines inbetween entries. 

Is that possible/Why do I get different results?

---

Sincerely,
~Gw

**Comment by Aidurber** [comment]
Hi @Gewerd-Strauss, 

I'm afraid I can't reproduce this. 

1. Are you using the Live Preview feature in Obsidian?
2. Can you reproduce this on the latest version 0.0.23? I'm thinking your issue is the same as #33 


My theory is some custom CSS rule. Can you tell me which theme you are using?

Below are 2 screenshots: 
- Default Obsidian Theme
- Things Theme
![2022-01-30 09_03_50-Debug - Obsidian v0 13 19](https://user-images.githubusercontent.com/5732291/151693491-7dbdd9ec-00ab-405c-b997-02fe1ffecd19.jpg)
![2022-01-30 09_04_03-Debug - Obsidian v0 13 19](https://user-images.githubusercontent.com/5732291/151693493-296c9b0a-d5f5-431f-81a0-36e73375b196.jpg)

**Comment by Gewerd-Strauss** [comment]
This is interesting.

I am using the  [Shimmering Focus](https://github.com/chrisgrieser/shimmering-focus) theme. Well, I am using four themes, depending on vault. However, when I created this issue I previously tested it on all my themes (including default), with the same problem happening across all vaults, themes and combinations. 

Now, I just did so again since you cannot reproduce it. 

No matter the theme/anything, I cannot reproduce it either, suddenly. 
![image](https://user-images.githubusercontent.com/80548781/151713457-91592c84-85a5-4979-a1d2-3262b2efda9e.png)
(Left is editor, right is preview)

I don't use any CSS snippets beyond the css of the Shimmering focus theme, but since this was happening independent of the active theme, I don't think it's the cause.

I have no idea what caused it, nor what resolved it again. We can close this issue. The next time I notice this happening, I'll make a copy of my vault and we'll where the problem lies.
